OsmAnd/src/net/osmand/plus/track/helpers/GPXDatabase.java

```diff
@@ -34,7 +39,7 @@ public class GPXDatabase {
 
 	public static final Log LOG = PlatformUtil.getLog(GPXDatabase.class);
 
-	private static final int DB_VERSION = 17;
+	public static final int DB_VERSION = 17;
```

Comment:
Why public?

Reply:
to be able check it when checking if recalculate analysis is needed